### ISSUE TITLE
Multi-sig on VS code

### ISSUE THREAD

**Issue by RedaKhyatti** [onflow/cadence#116]
Allowing multi-signature feature on VS code can help developers develop and test multi-sig transactions.

**Comment by turbolent** [comment]
Agreed, it would be nice to bring multi-signature transaction support to VS Code, just like we have it in the Playground.

This involves two parts: 
1. The VS Code extension: Add support for selecting multiple accounts
2. The language server: Sign transactions with all selected accounts

**Comment by turbolent** [comment]
@10thfloor are you working/planning to work on this?
